modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/WebSocketsManager.java

```diff
@@ -55,10 +56,14 @@
  * Web sockets manager.
  */
 @Service
+@EnableScheduling
```

Comment:
I think it is better to have this annotation on Application.

Reply:
Application is entry point. If you create another module that depends this - scheduling will work only after you add this anotation to new Application